amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -0,0 +1,36 @@
+@import 'variables';
+
+.table-issues {
+
+  .issue-banner {
+    border-radius: 4px;
+    box-shadow: 0px 0px 1px 0px rgba(0, 0, 0, 0.12), 0px 2px 3px 0px rgba(0, 0, 0, 0.16);
+    height: 40px;
+    margin: 8px 0;
+    padding: 8px 16px;
+
+    &:first-child {
+      margin-top: 24px;
+    }
+  }
+  .icon-red-triangle-warning  {
+    background: $red-triangle-warning;
+    margin: 0px 5px 0px 0px;
+  }
+  .table-issue-link {
+    margin: 0px 5px 0px 0px; 
+  }
+  .issue-title-name {
+    max-width: 390px; 
```

Comment:
n00b qq, for these css, does it normally get it from our UI designer?

Reply:
yes and no - something things i took straight from design (like colors), but sometimes things like default width and size of a row, font in designs is different than what's already in use. 

some things are represented in design but aren't specified (like the margins around the text).